tests/security/apparmor/aa_logprof.pm

```diff
@@ -39,7 +39,17 @@ sub run {
     my ($self) = @_;
     my $log_file = $apparmortest::audit_log;
     my $output;
-    my $aa_tmp_prof = "/tmp/apparmor.d";
+    my $aa_tmp_prof     = "/tmp/apparmor.d";
+    my $interactive_str = [
+        {
+            prompt => qr/\(A\)llow/m,
+            key    => 'a',
+        },
+        {
+            prompt => qr/\(S\)ave Changes/m,
+            key    => 's',
+        },
+    ];
 
     systemctl('stop nscd');
```

Comment:
Can you  add some comments here why stopping the nscd?

Reply:
> Can you add some comments here why stopping the nscd?

In order to generate some audit logs for testing "aa-logprof" (stop nscd before start it). Done